configure

```diff
@@ -6609,6 +6612,7 @@ enabled libopus           && {
     }
 }
 enabled libplacebo        && require_pkg_config libplacebo "libplacebo >= 4.192.0" libplacebo/vulkan.h pl_vulkan_create
+enabled libproresdecoder  && require_pkg_config libproresdecoder proresdecoder proresdecoder.h PROpenDecoder
```

Comment:
`require_pkg_config`: package config is present
`libproresdecoder`: library file name
`proresdecoder proresdecoder.h`: include header to look for
`PROpenDecoder`: actual frame decode function name